sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -2798,37 +2457,37 @@
         },
         "cloudToDeviceMethod": {
           "$ref": "#/definitions/CloudToDeviceMethod",
-          "description": "Required if jobType is cloudToDeviceMethod.\r\nThe method type and parameters."
+          "description": "The method type and parameters. This is required if the job type is cloudToDeviceMethod."
         },
         "updateTwin": {
           "$ref": "#/definitions/Twin"
         },
         "queryCondition": {
-          "description": "Required if jobType is updateTwin or cloudToDeviceMethod.\r\nCondition for device query to get devices to execute the job on",
+          "description": "The condition for devices to execute the job. This is required if the job type is updateTwin or cloudToDeviceMethod.",
           "type": "string"
         },
         "startTime": {
           "format": "date-time",
-          "description": "ISO 8601 date time to start the job",
+          "description": "The start date and time of the job in ISO 8601 standard.",
           "type": "string"
         },
         "maxExecutionTimeInSeconds": {
           "format": "int64",
-          "description": "Max execution time in secounds (ttl duration)",
+          "description": "The maximum execution time in secounds.",
           "type": "integer"
         }
       }
     },
     "CloudToDeviceMethod": {
-      "description": "Parameters to execute a direct method on the device",
+      "description": "The parameters to execute a direct method on the device.",
       "type": "object",
       "properties": {
         "methodName": {
-          "description": "Method to run",
+          "description": "The name of the method to execute.",
           "type": "string"
         },
         "payload": {
-          "description": "Payload",
+          "description": "The JSON-formatted direct method payload, up to 128kb in size.",
           "type": "object"
         },
         "responseTimeoutInSeconds": {
```

Comment:
Requesting description for responseTimeoutInSeconds and connectTimeoutInSeconds

Reply:
I thought we'd already provided descriptions for this.